src/Sentry.Unity.Editor.Tests/EditModeTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine.TestTools;
+
+namespace Sentry.Unity.Editor.Tests
+{
+    public sealed class EditModeTests
+    {
+        [UnityTest]
+        public IEnumerator OptionsReleaseField_LengthExceeds_CreatesError()
+        {
+            EditorHelpers.SetupScene("BugFarmScene");
+
+            // arrange
+            using var window = SentryWindow.OpenSentryWindow();
+
+            var validationErrors = new List<ValidationError>();
+            window.OnValidationError += error => validationErrors.Add(error);
+
+            // act
+            window.Options.Dsn = "qwerty";
+            yield return null; // mandatory, waiting for Unity to properly redraw and update inner states
+
+            // assert
+            Assert.AreEqual(1, validationErrors.Count);

```

Comment:
Ideally we'd validate the actual error too because it could easily have 1 error but be unrelated and test would pass.

Reply:
I agree.